templates/skel.html

```diff
@@ -4,6 +4,7 @@
 <meta charset="utf-8">
 <meta http-equiv="x-ua-compatible" content="ie=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<link rel="alternate" type="application/rss+xml" href="{{ config.base_url }}/rss.xml" title="blog.turbo.fish" />
```

Comment:
I generally don't use `config.base_url` to make local testing easier. Instead I just use site-relative links.

```suggestion
<link rel="alternate" type="application/rss+xml" href="/rss.xml" title="blog.turbo.fish" />
```